codecov.yml

```diff
@@ -0,0 +1,4 @@
+ignore: 
+  - "perun/collect/trace"
+  - "tests/test_collect"
```

Comment:
Fix: Please extract systemtap based test to test_tracer.py or something similar. Since test_collect contains tests for e.g. time, that can be well-tested in current environment.

Reply:
The tracer tests were extracted to a new separate module.